.spice/models/gas_fees/spec.py

```diff
@@ -0,0 +1,79 @@
+from typing import List, Optional
```

Comment:
Seems like this should be in a library that we provide? I don't think we want everyone that implements a model to have to redefine these.

No need to action now - just thinking out loud. Let's create an issue to track though.